Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_PlayerPresence.cs

```diff
@@ -17,18 +28,62 @@ public class VRTK_PlayerPresence : MonoBehaviour
         private float crouchMargin = 0.5f;
         private float lastPlayAreaY = 0f;
 
+        private bool isFalling = false;
+
+        public virtual void OnPresenceFallStarted(PlayerPresenceEventArgs e)
+        {
+            if (PresenceFallStarted != null)
+                PresenceFallStarted(this, e);
+        }
+
+        public virtual void OnPresenceFallEnded(PlayerPresenceEventArgs e)
+        {
+            if (PresenceFallEnded != null)
+                PresenceFallEnded(this, e);
+        }
+
+        protected PlayerPresenceEventArgs SetPlayerPhysicsEvent(float fallDistance)
```

Comment:
should this be protected? If it should then it's in the wrong location, the protected methods should appear after the public methods (but before the private methods)